tap_counter/ViewController.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  ViewController.swift
+//  tap_counter
+//
+//  Created by Michail on 9/10/16.
+//  Copyright © 2016 mishaparadising. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    var counter = 0;
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+        
+        tapCounter.text = String(counter);
+        
+        self.navigationItem.hidesBackButton = true;
+        
+        let resetButton = UIBarButtonItem(title: "Reset", style: UIBarButtonItemStyle.Bordered, target: self, action: Selector("resetButton"))
+        self.navigationItem.leftBarButtonItem = resetButton;
+        
+        let decreaseByOneButton = UIBarButtonItem(title: "Undo", style: UIBarButtonItemStyle.Bordered, target: self, action: Selector("decreaseByOne"))
+        self.navigationItem.rightBarButtonItem = decreaseByOneButton;
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+    @IBOutlet weak var tapCounter: UILabel!;
```

Comment:
Also, here is a VERY good article about inline docs http://nshipster.com/swift-documentation/. It is a bit long, I need you to learn the `MARK / TODO / FIXME` section and actively use it. You can skip everything else there for now.